meta-ostro/conf/distro/ostro.conf

```diff
@@ -236,6 +236,9 @@ PREFERRED_PROVIDER_virtual/java-initial-native = "cacao-initial-native"
 PREFERRED_PROVIDER_virtual/java-native = "cacao-native"
 PREFERRED_PROVIDER_virtual/javac-native = "ecj-bootstrap-native"
 
+# Java runtime
+PREFERRED_RPROVIDER_java2-runtime = "openjre-8"
```

Comment:
We are currently using openjdk-8: https://github.com/ostroproject/meta-ostro/blob/master/meta-ostro/recipes-core/packagegroups/packagegroup-java-jdk.bb

Whether we actually need to intall javac & friends is arguable however.